service/git/scanner.go

```diff
@@ -248,19 +250,20 @@ func (b *blobAdder) Fn(f *lookout.File) (bool, error) {
 
 	of, err := b.tree.File(f.Path)
 	if err != nil {
-		return true, err
+		log.Warningf("skipping - can not get file:'%v', %v", f.Path, err)
```

Comment:
can not -> cannot (here and 2 more instances below)

Reply:
addressed in e3daced